metis-statistics-data/src/main/java/eu/europeana/metis/HistoricalDataScript.java

```diff
@@ -0,0 +1,93 @@
+package eu.europeana.metis;
+
+import com.mongodb.client.MongoClient;
+import eu.europeana.metis.utils.Country;
+import eu.europeana.metis.model.Historical;
+import eu.europeana.metis.mongo.connection.MongoClientProvider;
+import eu.europeana.metis.config.MongoSDDao;
+import eu.europeana.metis.config.ConfigurationPropertiesHolder;
+import eu.europeana.metis.config.DataAccessConfigException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.CommandLineRunner;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class HistoricalDataScript implements CommandLineRunner {
+
+    private static final String COMMA_DELIMITER = ",";
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+
+
+    private final ConfigurationPropertiesHolder propertiesHolder;
+    private static final Logger LOGGER = LoggerFactory.getLogger(HistoricalDataScript.class);
+
+    public HistoricalDataScript(ConfigurationPropertiesHolder propertiesHolder) {
+        this.propertiesHolder = propertiesHolder;
+    }
+
+    @Override
+    public void run(String... args) throws DataAccessConfigException {
+
+        File folder = new File("metis-statistics-data/src/main/resources/historicalData");
+        File[] dataFiles = Objects.requireNonNull(folder.listFiles());
```

Comment:
my suggestion is to add a friendly message to this
```
Objects.requireNonNull(folder.listFiles()) 
```
to for example:
```
Objects.requireNonNull(folder.listFiles(),"a list of files required by <add description>");
```

Reply:
Wrote a slightly different message, let me know what you think

Reply:
is fine :smiley: